### ISSUE TITLE
✅ 🐞 Bug Report: "Apply to post content" pulls HTML markup and style from post content when using the Post Excerpt feature - [ED-10682]

### ISSUE THREAD

**Issue by LorenaFlyy** [elementor/elementor#22367]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Till Version 3.13 the loop grid showed no excerpt. Now, with the new version 3.13 it seemed fixed. But it isn't. If you start your blog post with a (styled) headline, the headline will be shown in the excerpt with its styles (that is: far to big) and not as usually without any difference to the text (paragraph) that follows. 

See the difference in the attached image: The first blog post starts with a headline (H2), the other two start with a paragraph. 

<img width="1331" alt="Bildschirmfoto 2023-05-09 um 09 31 25" src="https://user-images.githubusercontent.com/132489792/237025975-859f87fe-3fe0-4820-88b5-a9eebe86379c.png">

### Steps to reproduce

Use "Apply to post content" as described in Issue #20256

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: (Ubuntu) v5.7.37
	PHP Version: 8.0.28
	PHP Memory Limit: 256M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 200M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: xxx
	Home URL: xxx
	WP Multisite: No
	Max Upload Size: 200 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: de-DE
	Timezone: Europe/Berlin
	Admin Email: xxx
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: de_DE
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.13.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.13.0
		Author: Elementor.com

	Envato Elements
		Version: 2.0.12
		Author: Envato

	UpdraftPlus - Backup/Restore
		Version: 1.23.3
		Author: UpdraftPlus.Com, DavidAnderson

	WP Reset
		Version: 1.97
		Author: WebFactory Ltd


== Eigenschaften ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor-Experimente ==
	Optimierte DOM Ausgabe: Standardmäßig aktiviert
	Verbessertes Laden von Assets: Standardmäßig aktiviert
	Verbessertes Laden von CSS: Standardmäßig aktiviert
	Inline-Schriftarten-Symbole: Standardmäßig deaktiviert
	Verbesserungen der Zugänglichkeit: Standardmäßig aktiviert
	Zusätzliche benutzerdefinierte Breakpoints: Standardmäßig aktiviert
	admin_menu_rearrangement: Standardmäßig deaktiviert
	Flexbox Container: Aktiv
	Upgrade Swiper Library: Standardmäßig aktiviert
	Grid Container: Standardmäßig deaktiviert
	Default to New Theme Builder: Standardmäßig aktiviert
	Hello Theme Header & Footer: Inaktiv
	Editor Top Bar: Standardmäßig deaktiviert
	Startseiten: Inaktiv
	Nested Elements: Aktiv
	Lazy Load Background Images: Standardmäßig deaktiviert
	Page Transitions: Inaktiv
	Notes: Inaktiv
	Loop: Standardmäßig aktiviert
	Form Submissions: Standardmäßig aktiviert
	Scroll Snap: Standardmäßig aktiviert
	Menü: Standardmäßig deaktiviert


== Protokoll ==
	
JS: showing 5 of 5JS: 2023-05-03 10:56:30 [error X 1][https://xxx/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,utils&ver=6.2:2:31823] e.dispatchEvent is not a function 
JS: 2023-05-03 12:52:56 [error X 24][https://xxx/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:1081263] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-05-03 18:11:34 [error X 3][https://xxx/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:632658] Cannot read properties of undefined (reading 'isDesignable') 
JS: 2023-05-04 11:34:42 [error X 1][https://xxx/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.12.2:3:410136] Cannot read properties of null (reading 'history') 
JS: 2023-05-04 14:03:41 [error X 1][https://xxx/wp-admin/post.php?post=23&action=elementor#:7:51] Cannot read properties of undefined (reading 'tabs') 

PHP: showing 7 of 7PHP: 2023-05-03 13:42:13 [warning X 3][/www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-03 13:42:59 [error X 2][/www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on null in /www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(18, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /www/htdocs/w00ee9eb/xxx/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /www/htdocs/w00ee9eb/xxx/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /www/htdocs/w00ee9eb/xxx/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /www/htdocs/w00ee9eb/xxx/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-03 13:43:54 [warning X 33][/www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-03 19:02:56 [warning X 12][/www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::168] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-03 20:13:49 [warning X 99][/www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: /www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: /www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: /www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: /www/htdocs/w00ee9eb/xxxx/wp-content/plugins/elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2023-05-03 20:17:43 [notice X 1][/www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor/includes/template-library/sources/local.php::747] ob_end_clean(): Failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-05-04 20:56:18 [warning X 3][/www/htdocs/w00ee9eb/xxx/wp-content/plugins/elementor/app/modules/import-export/module.php::695] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 10 of 102023-05-09 09:11:59 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.0',
)]
2023-05-09 09:12:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.0',
)]
2023-05-09 09:15:35 [info] elementor::elementor_updater Started 
2023-05-09 09:15:35 [info] Elementor/Upgrades - _on_each_version Start  
2023-05-09 09:15:35 [info] Elementor/Upgrades - _on_each_version Finished 
2023-05-09 09:15:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.12.2',
  'to' => '3.13.0',
)]
2023-05-09 09:15:37 [info] elementor-pro::elementor_pro_updater Started 
2023-05-09 09:15:37 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-05-09 09:15:37 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-05-09 09:15:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.12.3',
  'to' => '3.13.0',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Kompatibel
	Envato Elements: Unkompatibel

== Elementor Pro - Compatibility Tag ==
	
	Envato Elements: Unkompatibel
				
````

</details>

**Comment by ursula-webeolia** [comment]
Hi,
I've seen the same and i would add that if there is something using html tags in the excerpt like links or bold texts it will mess up the layout.
![image](https://github.com/elementor/elementor/assets/84444093/3744ae8e-7d64-4cef-93ae-21497f064439)

**Comment by nicholaszein** [comment]
Hello @LorenaFlyy! 

Thank you for submitting your issue! 🙏 

We are aware of this issue and a fix is underway. We ask for patience as we work on implementing filters to sanitize the markup of the Post Excerpt widget.

Please understand that this is not part of issue #20256, and is something completely different that requires its own issue.

Kind regards

**Comment by nicholaszein** [comment]
Hi @LorenaFlyy!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.13.1**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
